train.py

```diff
@@ -132,8 +132,13 @@
     )
     parser.add_argument("--wandb-project-name", type=str, default="sb3", help="the wandb's project name")
     parser.add_argument("--wandb-entity", type=str, default=None, help="the entity (team) of wandb's project")
+    parser.add_argument("--yaml-file", type=str, default=None, help="the yaml file for updating script args")
     args = parser.parse_args()
 
+    # Update custom args with the custom yaml file ones if provided
```

Comment:
this should not be needed.
What I had in mind was to completely replace the `ars.yml` for instance by the file that is provided by the user